samples/net/lwm2m_client/overlay-ot.conf

```diff
@@ -31,7 +31,6 @@ CONFIG_OPENTHREAD_CHANNEL=26
 CONFIG_OPENTHREAD_NETWORKKEY="00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff"
 
 CONFIG_NET_CONFIG_MY_IPV6_ADDR="fdde:ad00:beef::1"
-CONFIG_NET_CONFIG_PEER_IPV6_ADDR="fdde:ad00:beef::2"
```

Comment:
This overlay should update `CONFIG_LWM2M_APP_SERVER`  not to change default behaviour.

Reply:
Fixed.